transport/src/androidUnitTest/kotlin/transport/core/MessageStoreTest.kt

```diff
@@ -99,6 +99,7 @@ internal class MessageStoreTest {
         )
 
         subject.prepareMessage(TestValues.TOKEN, "test message").run {
+            this.message as TextMessage
             assertThat(this.message.content).containsOnly(
```

Comment:
I would advise to test the cast as well: 
```suggestion
            assertThat(this.message).isInstanceOf(TextMessage::class.java)
            val textMessage = this.message as TextMessage
            assertThat(textMessage.content).containsOnly(
```

Reply:
And same for all the casts in tests in this PR